compair/static/test/features/step_definitions/view_user_accounts.js

```diff
@@ -0,0 +1,76 @@
+// Use the external Chai As Promised to deal with resolving promises in
+// expectations
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+
+var expect = chai.expect;
+
+var viewUserAccountsStepDefinitionsWrapper = function () {
+
+    this.Then("I should see '$count' LTI connections listed", function (count) {
+        browser.waitForAngular();
+        return expect(element.all(by.exactRepeater("lti_user in lti_users"))
+            .count()).to.eventually.eql(parseInt(count));
+    });
+
+    this.Then("I should see LTI connections with entries:", function (data) {
+        browser.waitForAngular();
+        var consumer_keys = data.hashes().map(function(item) {
+            return item.consumer_key;
+        });
+        var user_ids = data.hashes().map(function(item) {
+            return item.lti_user_id;
+        });
+
+        return expect(element.all(by.exactRepeater("lti_user in lti_users")
+            .column('lti_user.oauth_consumer_key')).getText()).to.eventually.eql(consumer_keys) &&
+                expect(element.all(by.exactRepeater("lti_user in lti_users")
+            .column('lti_user.lti_user_id')).getText()).to.eventually.eql(user_ids);
+    });
+
+    this.When("I select unlink for the first LTI connection listed", function () {
+        element.all(by.exactRepeater("lti_user in lti_users")).get(0)
+            .element(by.css('a[keyword="LTI user unlink link"]')).click();
```

Comment:
will need to update this after other change

Reply:
Sorry, I'm not clear what you want me to change here.

Reply:
https://github.com/ubc/compair/pull/655#discussion_r160538064